.github/workflows/build-and-upload-binaries-after-release.yaml

```
@@ -28,7 +28,7 @@ jobs:
           - platform: Linux ARM64
             os: ubuntu-latest
             artifact-path: compiler/cpp/cmake-build/bin
-            asset-name: thrift-${{ github.ref_name }}-linux-aarch64.tar.gz
+            asset-name: thrift-${{ github.ref_name }}-linux-arm.tar.gz
```

Не понял. Почему не
```suggestion
            asset-name: thrift-${{ github.ref_name }}-linux-arm64.tar.gz
```
?

---

https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
Архитектура просто `arm` судя по доке

---

Это примеры. `arm` эквивалентно `armv7l`, что я думаю не то что тебе нужно.

См. например https://hub.docker.com/_/alpine.

---

Хм, да